### PR TITLE
Removing cfg file from MmSupervisor package

### PR DESCRIPTION
**Pull request by kuqin12**
# Preface

Please ensure you have read the [contribution docs](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md) prior
to submitting the pull request. In particular,
[pull request guidelines](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md#pull-request-best-practices).

## Description

This change removed the temporary cfg file from MmSupervisorPkg because the real configuration should reside in the platform build to match the corresponding silicon release.

For each item, place an "x" in between `[` and `]` if true. Example: `[x]`.
_(you can also check items in the GitHub UI)_

- [x] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [ ] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

This was tested internally on a pipeline build.

## Integration Instructions

N/A

### CODE REVIEW

File: SpamPkg/Core/Init/StmInit.c [hunk 1]
```diff
@@ -801,7 +802,10 @@ BspInit (
   }
   // TODO
   // PcdSet64S(PcdPciExpressBaseAddress, mHostContextCommon.PciExpressBaseAddress);
-
+  mMyValue = PatchPcdGet8(PcdAuxBinHash);
```
Comment (kuqin12) [inline SpamPkg/Core/Init/StmInit.c:805]: Revert

File: SpamPkg/Tools/GenSpamArtifacts/gen_aux/src/main.rs [hunk 2]
```diff
@@ -113,6 +113,7 @@ pub fn main() -> Result<()> {
     // Add symbols from the individual modules
     let mut modules = debug_information.modules()?;
     while let Some(module) = modules.next()? {
+        println!("{:?}", module);
```
Comment (kuqin12) [inline SpamPkg/Tools/GenSpamArtifacts/gen_aux/src/main.rs:116]: Do we still need this?

**Review by apop5 — APPROVED** [review]

File: SpamPkg/Core/Init/StmInit.c (hunk 1 above)
Comment (Javagedes) [reply SpamPkg/Core/Init/StmInit.c]: Revert applied

File: SpamPkg/Tools/GenSpamArtifacts/gen_aux/src/main.rs (hunk 2 above)
Comment (Javagedes) [reply SpamPkg/Tools/GenSpamArtifacts/gen_aux/src/main.rs]: removed

**Review by Javagedes — APPROVED** [review]

**Pull request merged**